samples/webforms/embed-a-dashboard-into-an-app/README.md

```diff
@@ -0,0 +1,5 @@
+# Integrate a dashboard into an app
+Integrate a Power BI dashboard enables application developers to integrate Power BI dashboards from a user's power BI account by embedding an IFrame into an app, such as a mobile app or web app.
+
+To learn how to run this sample, see [Integrate a Power BI tile or report into an app](https://powerbi.microsoft.com/en-us/documentation/powerbi-developer-integrate-a-power-bi-tile-or-report/).
```

Comment:
We don't have a documentation for this sample yet, I will add a verification like this:
**A documentation for how to run this sample is coming soon. You can follow instructions for report/tile samples for now. See [Integrate a Power BI tile or report into an app](https://powerbi.microsoft.com/en-us/documentation/powerbi-developer-integrate-a-power-bi-tile-or-report/).**